apps/system/test/unit/call_forwarding_test.js

```diff
@@ -170,33 +170,51 @@ suite('system/callForwarding >', function() {
         window.asyncStorage.getItem.restore();
```

Comment:
Forgot to say, we should remove this test case `should set the icons state to false by default`.